packages/vm/src/runTx.ts

```diff
@@ -440,6 +447,51 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
     fromAccount.balance = BIGINT_0
   }
   await this.evm.journal.putAccount(caller, fromAccount)
+
+  const writtenAddresses = new Set<string>()
+  if (tx.supports(Capability.EIP7702EOACode)) {
+    const authorizationList = (<EIP7702CompatibleTx>tx).authorizationList
```

Comment:
```suggestion
    // Add contract code for authroity tuples provided by EIP 7702 tx
    const authorizationList = (<EIP7702CompatibleTx>tx).authorizationList
```

Would like something like this comment to give some minimal explanation of what's happening here for future devs